backend/cloud_inquisitor/plugins/types/resources.py

```diff
@@ -1279,3 +1279,82 @@ def update(self, data):
         updated = self.set_property('value', data['value'])
 
         return updated
+
+
+class VPC(BaseResource):
+    """VPC Object"""
+    resource_type = 'aws_vpc'
+    resource_name = 'VPC'
+
+    # region Object properties
+    @property
+    def cidr_v4(self):
+        """ Returns the IPv4 CIDR block associated with the VPC
+
+        Returns:
+            `str`
+
+        """
+        return self.get_property('cidr_v4').value
+
+    @property
+    def state(self):
+        """ Returns the current state of the VPC (pending/available)
+
+        Returns:
+            `str`
+
+        """
+        return self.get_property('state').value
+
+    @property
+    def vpc_flow_logs_status(self):
+        """ Returns the configured state of VPC Flow Logs
+
+        Returns:
+            `str`
+
+        """
+        return self.get_property('vpc_flow_logs_status').value
+
+    def vpc_flow_logs_log_group(self):
+        """ Returns the CloudWatch Log Group associated with the VPC Flow Logs
+
+        Returns:
+            `str`
+
+        """
+        return self.get_property('vpc_flow_logs_log_stream').value
+
+    # end of region
+
+    def update(self, data, client_data):
+        """Updates the object information based on live data, if there were any changes made. Any changes will be
+        automatically applied to the object, but will not be automatically persisted. You must manually call
+        `db.session.add(vpc)` on the object.
+
+        Args:
+            data (bunch): Data fetched from AWS API
```

Comment:
You are missing the documentation for the client_data argument

Reply:
Ah quite right...in fact what's being passed here is actually just properties for the object so I'm going to rename this and document

Reply:
@bunjiboys : lmk if that looks better.